Dockerfile

```diff
@@ -14,11 +14,17 @@ COPY . md_source
 # into static HTML in the "target" directory using Jekyll
 # then nuke the md_source directory.
 
+# Process this as an Edge release
+ENV EDGE 1
+
 ## Branch to pull from, per ref doc
-ENV ENGINE_BRANCH="17.03.x"
+## To get master from svn the svn branch needs to be 'trunk'. To get a branch from svn it needs to be 'branches/branchname'
+ENV ENGINE_SVN_BRANCH="branches/17.04.x"
+ENV ENGINE_BRANCH="17.04.x"
+ENV DISTRIBUTION_SVN_BRANCH="branches/release/2.5"
```

Comment:
Any reason for including `branches/` into this var, or is it to allow using `commit`?